data/src/message.rs

```diff
@@ -544,12 +544,24 @@ impl Limit {
     }
 }
 
-fn is_action(text: &str) -> bool {
-    text.starts_with("\u{1}ACTION ") && text.ends_with('\u{1}')
+pub fn ctcp_command(text: &str) -> Option<&str> {
+    let command = text
+        .strip_suffix('\u{1}')
+        .unwrap_or(text)
```

Comment:
Is the suffix optional? Why do we not require it?

Reply:
I was going by [this CTCP draft](https://raw.githubusercontent.com/DanielOaks/irc-rfcs/master/dist/draft-oakley-irc-ctcp-latest.txt), which is the most recent draft I found linked from <https://modern.ircdocs.horse/ctcp>.  It says,
```
   The final CTCP delimiter SHOULD be sent on outgoing messages for
   compatibility reasons, but software MUST accept incoming messages
   which lack it.
```
But I'm not particularly attached to adhering to the draft, it's just the closest thing I could find to a standard.

Reply:
Makes sense 👍